lib/rules/font-family-name-quotes/index.js

```diff
@@ -83,11 +85,16 @@ const makeMutableFontFamilies = (fontFamilies, decl) => {
 	const mutableNodes = [];
 
 	fontFamilies.forEach((fontFamily, idx) => {
+		const quote = 'quote' in fontFamily && fontFamily.quote;
```

Comment:
[note] The [`StringNode#quote`](https://github.com/TrySound/postcss-value-parser/blob/d4a582692533fc87cbab348136e1dbea365d5a94/lib/index.d.ts#L62-L65) property holds an actual quote like `"` or `'`.